MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs

```diff
@@ -537,6 +542,10 @@ private void SignupEmailInput(SessionBase session)
 
             //Create the user in the database
             _accountRepository.InsertAccount(session.Username, session.Password, session.Email);
+            
+            //Add The User to the BBS Btrieve User Database
+            var _accountBtrieve = _globalCache.Get<BtrieveFileProcessor>("ACCBB-PROCESSOR");
+            _accountBtrieve.Insert(new UserAccount { userid = Encoding.ASCII.GetBytes(session.Username), psword = Encoding.ASCII.GetBytes("<<HASHED>>") }.Data);
```

Comment:
is `<<HASHED>>` supposed to be legit or just a hidden value that modules shouldn't be messing with anyways?

Reply:
BBSUSR.DAT and the `usrprt` struct store the password in plaintext. Because MBBSEmu uses hashed passwords, they're both too long and incompatible with this field. The `ACCBB` pointer (as far as we've come across) is for looking up if a given username is present on the system. One game uses it to make sure the character name you select isn't already a user on the BBS (to prevent confusion). 

I'd wager other than username lookup, we're probably safe here.